public/app/core/components/NavBar/NavBarMenu.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useRef } from 'react';
+import { GrafanaTheme2, NavModelItem } from '@grafana/data';
+import { CustomScrollbar, Icon, IconButton, IconName, useTheme2 } from '@grafana/ui';
+import { FocusScope } from '@react-aria/focus';
+import { useOverlay } from '@react-aria/overlays';
+import { css } from '@emotion/css';
+import { NavBarMenuItem } from './NavBarMenuItem';
+
+export interface Props {
+  activeItem?: NavModelItem;
+  navItems: NavModelItem[];
+  onClose: () => void;
+}
+
+export function NavBarMenu({ activeItem, navItems, onClose }: Props) {
```

Comment:
don't think we should call it `MobileNavBar` because it's not _just_ used on mobile, it's also used as a mega menu when the feature toggle is on. `NavBarMenu` is definitely poor though.

we need something that conveys it shows the **full** `navTree`. `NavMenuComplete`? `NavMenuExpanded`? any suggestions? 🤔

Reply:
You're right, then I think its ok for now. 👍